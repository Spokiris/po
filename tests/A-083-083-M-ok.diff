*** tests/A-083-083-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-083-083-M-ok.outhyp	2023-10-27 19:48:54.203018026 +0100
***************
*** 32,40 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 1;1|
! 1;2|
! 1;3|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 32,43 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 
! 
! 
! 
! 
! 
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 44,52 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: 1;1|
! 1;2|
! 1;3|
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 47,53 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: xxl.core.CutBuffer@3cd1a2f1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
