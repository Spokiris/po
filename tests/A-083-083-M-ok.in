4
3
5;1:5;3
6
1;1:1;3
1
1;1:1;3
7
0
0
