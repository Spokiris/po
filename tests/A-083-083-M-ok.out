Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 4
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 3
Especifique a gama (startline;startcol:endline;endcol): 5;1:5;3
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 6
Especifique a gama (startline;startcol:endline;endcol): 1;1:1;3
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 1
Especifique a gama (startline;startcol:endline;endcol): 1;1:1;3
1;1|
1;2|
1;3|
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 7
1;1|
1;2|
1;3|
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 0
